### PR TITLE
Update Readme

### PR DESCRIPTION
**Pull request by alexandern17**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,24 +22,24 @@
...
     + [Send a response code](#send-a-response-code)
   * [Identity sync](#identity-sync)
     + [Identity sync with Thunderhead ONE or Salesforce Interaction Studio](#identity-sync-with-thunderhead-one-or-salesforce-interaction-studio)
     + [Identity sync with Thunderhead ONE or Salesforce Interaction Studio and your web touchpoint](#identity-sync-with-thunderhead-one-or-salesforce-interaction-studio-and-your-web-touchpoint)
   * [Ability to whitelist identity transfer links](#ability-to-whitelist-identity-transfer-links)
   * [Ability to blacklist identity transfer links](#ability-to-blacklist-identity-transfer-links)
   * [Disable automatic identity transfer](#disable-automatic-identity-transfer)
-    + [Send properties for a URL scheme](#send-properties-for-a-url-scheme)
+    + [Send Properties for a URL scheme](#send-properties-for-a-url-scheme)
     + [Append a ‘one-tid’ parameter to a `NSURL` to facilitate identity transfer](#append-a-one-tid-parameter-to-a-nsurl-to-facilitate-identity-transfer)
```
Comment (skhozin) [inline README.md:43]: wrong quotes

File: README.md [hunk 2]
```diff
@@ -22,24 +22,24 @@
...
+    + [Send an Interaction request](#send-an-interaction-request)
+    + [Send an Interaction request and retrieve the response](#send-an-interaction-request-and-retrieve-the-response)
+  * [Retrieve a response for an automatically triggered Interaction request](#retrieve-a-response-for-an-automatically-triggered-interaction-request)
     + [Retrieve a response for other instances](#retrieve-a-response-for-other-instances)
   * [Send Properties to Thunderhead ONE or Salesforce Interaction Studio](#send-properties-to-thunderhead-one-or-salesforce-interaction-studio)
-    + [Send properties to a base touchpoint](#send-properties-to-a-base-touchpoint)
-    + [Send an Interaction request with properties](#send-an-interaction-request-with-properties)
-    + [Send an Interaction request with properties and a completion block](#send-an-interaction-request-with-properties-and-a-completion-block)
-    + [Send properties to an Interaction](#send-properties-to-an-interaction)
+    + [Send Properties to a base touchpoint](#send-properties-to-a-base-touchpoint)
```
Comment (skhozin) [inline README.md:31]: Touchpoint

File: README.md [hunk 3]
```diff
@@ -22,24 +22,24 @@
...
+    + [Send Properties to an Interaction](#send-properties-to-an-interaction)
+    + [Send an Interaction request with Properties](#send-an-interaction-request-with-properties)
+    + [Send an Interaction request with Properties and retrieve the response](#send-an-interaction-request-with-properties-and-retrieve-the-response)
     + [Send a response code](#send-a-response-code)
   * [Identity sync](#identity-sync)
     + [Identity sync with Thunderhead ONE or Salesforce Interaction Studio](#identity-sync-with-thunderhead-one-or-salesforce-interaction-studio)
     + [Identity sync with Thunderhead ONE or Salesforce Interaction Studio and your web touchpoint](#identity-sync-with-thunderhead-one-or-salesforce-interaction-studio-and-your-web-touchpoint)
```
Comment (skhozin) [inline README.md:38]: Touchpoint

File: README.md [hunk 4]
```diff
@@ -22,24 +22,24 @@
...
   * [Ability to whitelist identity transfer links](#ability-to-whitelist-identity-transfer-links)
   * [Ability to blacklist identity transfer links](#ability-to-blacklist-identity-transfer-links)
   * [Disable automatic identity transfer](#disable-automatic-identity-transfer)
-    + [Send properties for a URL scheme](#send-properties-for-a-url-scheme)
+    + [Send Properties for a URL scheme](#send-properties-for-a-url-scheme)
```
Comment (skhozin) [inline README.md:42]: `URL`

**Comment by alexandern17** [comment]
@skhozin updated

**Pull request merged**